gamedb.yaml

```diff
@@ -4019,6 +4019,19 @@
   id: 860950
   status: verified
 
+- name: "Marvel's Guardians of the Galaxy"
+  platform: epic-store
+  id: 63a665088eb1480298f1e57943b225d8
+  status: unsupported
+  notes:
+    - Launches when installed via Heroic Games Launcher, but has mediocre controller support (tested with the Steam Controller)
```

Comment:
This note is not really relevant and seems out of place and confusing. Please remove.

Reply:
O.K. I added this in case someone still wants to try to know what he's up to.